### PR TITLE
Add new state for buttons

### PR DESCRIPTION
**Pull request by IgnacioRodrigues**
New state for buttons:
 - Secondary button green:
Screenshot (7)
- Secondary button brown:
Screenshot

### CODE REVIEW

**Comment by andresmoschini** [comment]
it is ok for me, @mvillaseco @GusBaamonde what do you think?

File: assets/scss/modules/_buttons.scss [hunk 1]
```diff
@@ -89,6 +89,30 @@
...
+      }
+
+      &brown {
+        background: transparent;
+        border: 1px solid $dp-color-brown;
+        color: $dp-color-brown;
+
+        &:hover {
+          background: $dp-color-brown-brightness;
+          color: $dp-color-brown;
```
Comment (fcoronelmakingsense) [inline assets/scss/modules/_buttons.scss]: maybe don't need this. it's the font color define in the css class

Comment (IgnacioRodrigues) [reply assets/scss/modules/_buttons.scss]: It's true, thanks @fcoronelmakingsense

**Comment by GusBaamonde** [comment]
Muy bueno Nacho!

**Pull request merged**